lms/static/sass/discussion/elements/_editor.scss

```diff
@@ -139,6 +139,44 @@
   > form > input[type="file"] {
     margin-bottom: 18px;
   }
+
+  .field-group .field .field-hint {
+    @include margin-left(0);
+    width: 100%;
+  }
+
+  .field-input-label {
+    font-size: $forum-base-font-size;
+  }
+
+  .input-text {
+    width: 62%;
```

Comment:
62% width here (and on L162 below) is a little arbitrary, can you add either a comment showing how you got to this math or explicitly do it with `calc()` here?